app/assets/javascripts/discourse/app/controllers/sidebar-section-form.js

```diff
@@ -77,14 +77,36 @@ class SectionLink {
     return this.name === undefined || this.validName ? "" : "warning";
   }
 
+  get internal() {
+    return (
+      this.value.startsWith(this.protocolAndHost) || this.value.startsWith("/")
+    );
+  }
+
+  get validExternal() {
```

Comment:
```suggestion
  #validExternal() {
```

I think this can  just be a private function and doesn't need to be a getter.